rust-src/concordium_base/src/hashes.rs

```diff
@@ -26,11 +26,6 @@ pub enum UpdateSignMarker {}
 /// Used as a phantom type to indicate a hash is a block state hash.
 pub enum StateMarker {}
 
```

Comment:
Since these are public, either reexport them from `concordium_contract_common` here or update the changelog of `concordium-base` with these being moved

Reply:
I think they are re-exported already.
```
pub use concordium_contracts_common::hashes::*;
```

Reply:
So there is no change in base if I'm not mistaken.